bin/generate_pages

```diff
@@ -45,7 +45,7 @@ function list_one_criterion {
 
 }
 
-list_one_criterion psnr x264 x264_base vp8 vp9 x265 openh264
+list_one_criterion psnr x264 x264_base vp8 vp9 x265 openh264 libavc
```

Comment:
Would it make sense to switch over to alpha sorting the codecs here, as the list is growing?